CMakeLists.txt

```diff
@@ -20,8 +20,8 @@ project(cfsd18-perception-detectcone)
 ################################################################################
```

Comment:
You might want to change the header author to Cfsd instead of Christian :D @chrberger .

Reply:
Yes, please :-)